cli/src/cli/command/pipeline/commands.py

```
@@ -0,0 +1,67 @@
+
+    except Exception as ex:
+        rprint(f"[red]❌ Error: {ex}[/red]")
+        if verbose:
+            rprint(f"[dim]{traceback.format_exc()}[/dim]")
+        raise Exit(1) from ex
+
+
+@pipeline_app.command("validate")
+def validate_pipeline(
+    config_path: Annotated[
+        Path, Argument(help="Path to YAML configuration", exists=True)
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)